core/templates/pages/admin-page/config-tab/admin-config-tab.directive.ts

```diff
@@ -40,6 +40,7 @@ angular.module('oppia').directive('adminConfigTab', [
         var ctrl = this;
         ctrl.isNonemptyObject = function(object) {
           var hasAtLeastOneElement = false;
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can we find some other way to check it there is some property in object rather than using for loop and than adding a eslint disable statement?

Reply:
Done.